pkg/odo/cli/component/push.go

```diff
@@ -245,6 +278,8 @@ func NewCmdPush(name, fullName string) *cobra.Command {
 	pushCmd.Flags().StringVarP(&po.componentContext, "context", "c", "", "Use given context directory as a source for component settings")
```

Comment:
small suggestion, can you add autocompletion for `--context` flag, it can be different PR also.

```
completion.RegisterCommandFlagHandler(pushCmd, "context", completion.FileCompletionHandler)
```

Reply:
sure